feat: 8.9 additions

NathanWalker opened the pull request.

vakrilov commented on navigation.md:
```diff
@@ -34,40 +34,37 @@ NativeScript apps consist of pages which represent the separate application scre
-You can create the pages for your user interface separately from your business logic. 
+You can create the pages for your user interface separately from your code. 
+
+To apply this approach, you need to create a separate `XML` file for each page to hold the layout of the page. Thus your code will be in a `JS` or a `TS` file. The names of the `XML` and the `JS` or `TS` file must match.
```
Suggestion: drop the "you need to" and "separate" - less words is better

vakrilov commented on navigation.md:
```diff
@@ -34,40 +34,37 @@ NativeScript apps consist of pages which represent the separate application scre
-**Create page when executing business logic**
+**Create a page in code**
 
 You can create the page inside your business logic.
```
"business logic" - better use only "code"
For example: "Create a page entirely from code"

vakrilov commented on navigation.md:
```diff
@@ -34,40 +34,37 @@ NativeScript apps consist of pages which represent the separate application scre
+**Create a page in XML**
 
-You can create the pages for your user interface separately from your business logic. 
+You can create the pages for your user interface separately from your code. 
```
Suggestion: "You can define the UI declaration and the code for the page separately."

vakrilov commented:
:+1:

NathanWalker committed:
feat: flavor snippets wip

NathanWalker committed:
Merge remote-tracking branch 'origin/main' into feat/flavor-snippets

NathanWalker committed:
chore: cleanup template

NathanWalker committed:
chore: add shared assets

NathanWalker committed:
chore: cleanup dialogs

NathanWalker committed:
feat: angular flavor wip

Commit:
feat: angular flavor

NathanWalker committed:
Merge remote-tracking branch 'origin/main' into feat/flavor-snippets

NathanWalker committed:
feat: vue 3 flavor wip

mbflores committed:
Updates: Vue Flavor

DatePicker
HtmlView
Image

mbflores committed:
Flavors Vue Update

ListPicker
Palceholder
Progress
Scrollview
SegmentedBar

Commit:
feat: angular flavor - ActionBar and TabView fixes

mbflores committed:
Updates: Vue Flavor

Slider
Switch
TabView
TextField
TextView
TimePicker
WebView

mbflores committed:
Flavor: Vue

Fix ActionBar -- must be Actionbar to work, temporary solution
Activity Indicator
Dialogs

mbflores committed:
Flavor: Vue

Fix Actionbar, it doesn't load if the foldername is ActionBar, seems reserve word
Fix Label, it doesn't load if the foldername is Label, seems reserve word
Fix ListView
Fix PlaceHolder

mbflores committed:
Fix Webview
Add value-binding for Progress to have interactive sample
Simplify binding for ListPicker
clean label
clean image
simplify binding for htmlview
added model binding for DatePicker to have interactive example
Button- added tap interaction
added package for android

mbflores committed:
Flavor: Svelte

WIP
bootstrapping of svelte flavor
ActionBar
ActivityIndicator
Button

mbflores committed:
Flavors: Svelte Update

Datepicker
DIalogs
Update titles of actionbar

mbflores committed:
Flavor Update: Svelte

HtmlView
Image
Label
ListPicker
ListView
Placeholder
Progress
ScrollView

mbflores committed:
Flavor Update: Svelte

SearchBar
SegmentedBar
Slider
Switch
TabView
TextField
TextView
TimePicker
WebView

mbflores committed:
Finalized Svelte Flavor

mbflores committed:
Flavor: SolidJS

mbflores committed:
WIP Solid

mbflores committed:
Flavor Update: Solid

remove screenthree, and remove usage of globals. and collection

mbflores committed:
Flavor Update: SolidJS

bootstrap all pages, with default, and simple content

mbflores committed:
Flavor Update: SolidJS

actionbar
activityindicator
button
datepicker
dialogs
htmlview
label
listpicker
listview

mbflores committed:
Flavor Update: SolidJS

Datepicker
Dialogs
HTMLVIew
image
Label
ListPicker
listview
Placeholder
progress
scrollview
searchbar
segmentedbar
slider
switch
tabview
textfield
textview
timepicker
webview

mbflores committed:
Flavor Update: React

Bootstrap template needed for React, Starting project for Flavor Samples

mbflores committed:
Flavor Update: React

ActionBar
ActivityIndicator
Button
DatePicker
Dialogs
HtmlView
Image
Label
ListPicker
Placeholder
Progress
ScrollView
SearchBar
SegmentedBar
Slider
Switch
TabView
TextField
TextView
TimePicker
WebView

NathanWalker committed:
Merge remote-tracking branch 'origin/main' into feat/flavor-snippets

NathanWalker committed:
feat: add snippets to ui guide component pages

NathanWalker committed:
Merge remote-tracking branch 'origin/main' into feat/flavor-snippets

rigor789 committed:
feat: add tsx region support for snippets

NathanWalker committed:
feat: updated angular sample to 19.1

rigor789 committed:
fix: clean up snippets

rigor789 committed:
feat: new tabs

NathanWalker committed:
feat: 8.9 additions

NathanWalker committed:
chore: format

NathanWalker committed:
feat(config): ios.NativeSource option (#161)

NathanWalker committed:
feat: ns widget ios

NathanWalker committed:
format

NathanWalker committed:
ci: build

NathanWalker committed:
Merge remote-tracking branch 'origin/main' into feat/flavor-snippets

NathanWalker committed:
Merge branch 'main' into feat/flavor-snippets

NathanWalker committed:
Merge branch 'main' into feat/8.9-core-additions

NathanWalker committed:
chore: bump vitepress

NathanWalker committed:
fix: build

NathanWalker committed:
chore: format

NathanWalker committed:
Merge branch 'feat/8.9-core-additions' into feat/flavor-snippets

NathanWalker committed:
feat: flavor snippets (#151)

NathanWalker committed:
chore: widget docs

NathanWalker committed:
chore: docs tabs

NathanWalker committed:
feat: flavor tabs

NathanWalker committed:
ci: format

NathanWalker committed:
ci: cleanup

NathanWalker committed:
chore: update cli to bash for syntax highlighting

NathanWalker committed:
feat: widget video

NathanWalker committed:
feat: solid vibes snack

NathanWalker committed:
chore: updates

NathanWalker committed:
Merge branch 'main' into feat/8.9-core-additions

NathanWalker committed:
chore: yarn

NathanWalker committed:
chore: asset path

NathanWalker committed:
ci: build
